src/controller.cc

```diff
@@ -244,6 +247,9 @@ void Controller::IssueCommand(const Command &cmd) {
         // if there are multiple reads pending return them all
         while (num_reads > 0) {
             auto it = pending_rd_q_.find(cmd.hex_addr);
+            it->second.issue_cycle = clk_;
+    	    simple_stats_.AddValue("queuing_latency", it->second.issue_cycle - it->second.added_cycle);
```

Comment:
This is the time in the transaction queue. I would update the name to "transaction queuing latency" or something like that.